lib/Charts/ChartRenderer.js

```diff
@@ -118,23 +158,34 @@ class ChartRenderer {
   }
 
   static render(container, state) {
+    state.dblClickTimer = false;
     if (!defined(state.data)) {
       return;
     }
 
+    // We want points to appear on top of everything else
+    state.data.sort(function(a, b) {
```

Comment:
This sort isn't guaranteed to the stable. MDN says it's not on IE, Edge, and possibly Safari: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort#Browser_compatibility
But because you're returning -1 for all non-moment-points, it'll change the relative order even in browsers that do a stable sort.

Reply:
Have replaced with a simple for loop